universal/include/userver/utils/string.hpp

```diff
@@ -0,0 +1,63 @@
+
+#pragma once
+#include <algorithm>
+#include <string>
+USERVER_NAMESPACE_BEGIN
+
+namespace utils {
+
+template <std::size_t Size = 1>
+struct ConstexprString {
+  static constexpr auto kSize = Size;
+  std::array<char, Size> contents;
```

Comment:
Let's call it `contents_` and define as follows:

```cpp
/// @cond
// Implementation detail.
char contents_[Size]{};
/// @endcond
```

Reply:
But why not std::array ?

Reply:
`<array>` brings in `<algorithm>` in typical stdlib implementations, and that's a very heavy header. We try to minimize the usage of `<array>` in public headers